.github/CODEOWNERS

```diff
@@ -147,6 +147,7 @@
 /traefik/                                @renaudhager
 /trino/                                  @ndrluis
 /twenty_forty_eight/                     @DataDog/apps-sdk
+/twingate/                               @chenbishop partnerships@twingate.com
```

Comment:
Could you also update this section? https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L167

Reply:
I have proposed the change for this, could you review and accept please?

https://github.com/DataDog/integrations-extras/pull/1590

Reply:
Thanks 🙂 Could you include the change in this PR so we have only one PR that includes everything?

Also you will need to add the `manifest.json` file.

Reply:
Hi @FlorentClarret - I have updated CODEOWNERS. What wasn't clear to us was that we needed 2x Twingate entries which we have now in this PR. Also, `manifest.json` is included in this PR already - can you clarify what is wrong with it?

Reply:
Oh sorry if this was not clear. Having twingate mentioned in the other other section too allow us to automatically ping our documentation team if you open a PR and update either the README or the manifest file so they can take a look. You just need to also add this line in the CODEOWNERS file: 

```
/twingate/manifest.json                      @emrul partnerships@twingate.com @DataDog/documentation
```

And we'll be good to go :)

Reply:
@FlorentClarret - that makes much more sense now!! Thank you - I have updated the `CODEOWNERS` to also include a reference to manifest - Let's gooooo 💪